pkg/gds/models/v1/models_test.go

```diff
@@ -26,6 +27,32 @@ func TestVASPExtra(t *testing.T) {
 	token, err = GetAdminVerificationToken(vasp)
 	require.NoError(t, err)
 	require.Equal(t, "pontoonboatz", token)
+
+	// Attempt to append to verification change log
+	entry := &VerificationChangeLogEntry{
+		Timestamp:    time.Now().Format(time.RFC3339),
+		CurrentState: uint32(pb.VerificationState_VERIFIED),
+		Description:  "automated",
+		Source:       "pontoon@boatz.com",
+	}
+	err = AppendVerificationChangeLog(vasp, entry)
+	require.NoError(t, err)
+
+	// Should be able to fetch the new change log
+	changeLog, err := GetVerificationChangeLog(vasp)
+	require.NoError(t, err)
+	require.Len(t, changeLog, 1)
+	//TODO: Is there a better way to compare protobuf objects?
```

Comment:
If you want to make sure the protocol buffers have the same serial representation (e.g. all the fields are equal) you can use [`proto.Equal`](https://pkg.go.dev/google.golang.org/protobuf/proto#Equal)